### DISCUSSION TITLE
How to allow external mysql access

### DISCUSSION THREAD

**Discussion by Pablojpas** [EasyEngine/easyengine#1557] [Support Requests]
Hi guys, I would like you to help me enable the MySql external connection as the EasyEngine tutorial seems to be out of date and I can't get it to work. Can anyone guide me please? 

I get an error where I enable the bind-address in the my.conf and no site in wordpress can connect to the database.

Thanks

**Comment by mrrobot47** [comment]
https://community.easyengine.io/t/solved-eev4-macos-mojave-and-managing-db-from-sequelpro-or-tableplus/12152/5

```bash
cd /opt/easyengine/services
```
Update `docker-compose.yml` and add ports to global-db:
```yml
  global-db:
    ports:
     - "3306:3306"
    image: easyengine/mariadb:v4.1.3
```
Recreate DB container:

```bash
cd /opt/easyengine/services && docker-compose up -d global-db
```
Now, access it from Workbench/externallu using localhost with port 3306. You will also get root creds of DB from this docker-compose.yml file.

**Reply by ssuess** [reply]
Any way to prevent this from being overwritten on upgrade?

**Reply by dhsathiya** [reply]
Nope. You might try EasyEngine's [custom docker-compose feature](https://easyengine.io/handbook/customising-docker-compose-stack-in-easyengine-v4/). 
_⚠️ Do it on your own risk._
